Build failing on sfcgal master branch missing nlohmann_json

The buildenv building has been failing since Oct 17th with SFCGAL compile complaining about missing nlohmann_json

```
#10 [ 6/15] RUN git clone --depth 1 --branch master https://gitlab.com/sfcgal/SFCGAL.git &&      cd SFCGAL &&      mkdir cmake-build &&      cd cmake-build &&      cmake -DCGAL_DIR="/src/CGAL" -DCMAKE_PREFIX_PATH=/src/CGAL .. &&      make -j4 &&      make install &&      cd /src && rm -rf SFCGAL
#10 0.905 Cloning into 'SFCGAL'...
#10 2.105 CMake Deprecation Warning at CMakeLists.txt:1 (cmake_minimum_required):
#10 2.105   Compatibility with CMake < 3.10 will be removed from a future version of
#10 2.105   CMake.
#10 2.105 
#10 2.105   Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
#10 2.105   to tell CMake that the project requires at least <min> but has been updated
#10 2.105   to work with policies introduced by <max> or earlier.
#10 2.105 
#10 2.105 
#10 2.285 -- The C compiler identification is GNU 14.2.0
#10 2.518 -- The CXX compiler identification is GNU 14.2.0
#10 2.543 -- Detecting C compiler ABI info
#10 2.768 -- Detecting C compiler ABI info - done
#10 2.812 -- Check for working C compiler: /usr/bin/cc - skipped
#10 2.814 -- Detecting C compile features
#10 2.816 -- Detecting C compile features - done
#10 2.849 -- Detecting CXX compiler ABI info
#10 3.081 -- Detecting CXX compiler ABI info - done
#10 3.103 -- Check for working CXX compiler: /usr/bin/c++ - skipped
#10 3.104 -- Detecting CXX compile features
#10 3.106 -- Detecting CXX compile features - done
#10 3.121 -- Setting build type to 'Release' as none was specified.
#10 3.131 -- Using header-only CGAL
#10 3.134 -- Targeting Unix Makefiles
#10 3.134 -- Using /usr/bin/c++ compiler.
#10 3.139 -- Found GMP: /usr/lib/x86_64-linux-gnu/libgmp.so
#10 3.141 -- Found MPFR: /usr/lib/x86_64-linux-gnu/libmpfr.so
#10 3.150 CMake Warning (dev) at /src/CGAL/lib/cmake/CGAL/CGAL_SetupBoost.cmake:20 (find_package):
#10 3.150   Policy CMP0167 is not set: The FindBoost module is removed.  Run "cmake
#10 3.150   --help-policy CMP0167" for policy details.  Use the cmake_policy command to
#10 3.150   set the policy and suppress this warning.
#10 3.150 
#10 3.150 Call Stack (most recent call first):
#10 3.150   /src/CGAL/lib/cmake/CGAL/CGAL_SetupCGALDependencies.cmake:48 (include)
#10 3.150   /src/CGAL/lib/cmake/CGAL/CGALConfig.cmake:168 (include)
#10 3.150   CMakeLists.txt:102 (find_package)
#10 3.150 This warning is for project developers.  Use -Wno-dev to suppress it.
#10 3.150 
#10 3.158 -- Found Boost: /usr/lib/x86_64-linux-gnu/cmake/Boost-1.83.0/BoostConfig.cmake (found suitable version "1.83.0", minimum required is "1.66")
#10 3.159 -- Boost include dirs: /usr/include
#10 3.159 -- Boost libraries:    
#10 3.169 -- Performing Test CMAKE_HAVE_LIBC_PTHREAD
#10 3.406 -- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
#10 3.409 -- Found Threads: TRUE
#10 3.409 -- Using gcc version 4 or later. Adding -frounding-math
#10 3.410 -- CGAL 5.6 found
#10 3.425 -- Copying headers from /src/SFCGAL/src to /src/SFCGAL/cmake-build/include/SFCGAL
#10 3.475 -- Boost_USE_STATIC_LIBS=OFF
#10 3.475 -- Boost_USE_MULTITHREAD=ON
#10 3.495 CMake Error at CMakeLists.txt:180 (find_package):
#10 3.495   By not providing "Findnlohmann_json.cmake" in CMAKE_MODULE_PATH this
#10 3.495   project has asked CMake to find a package configuration file provided by
#10 3.495   "nlohmann_json", but CMake did not find one.
#10 3.495 
#10 3.495   Could not find a package configuration file provided by "nlohmann_json"
#10 3.495   (requested version 3.11) with any of the following names:
#10 3.495 
#10 3.495     nlohmann_jsonConfig.cmake
#10 3.495     nlohmann_json-config.cmake
#10 3.495 
#10 3.495   Add the installation prefix of "nlohmann_json" to CMAKE_PREFIX_PATH or set
#10 3.495   "nlohmann_json_DIR" to a directory containing one of the above files.  If
#10 3.495   "nlohmann_json" provides a separate development package or SDK, be sure it
#10 3.495   has been installed.
#10 3.495 
#10 3.495 
#10 3.496 -- Configuring incomplete, errors occurred!
#10 ERROR: process "/bin/sh -c git clone --depth 1 --branch ${SFCGAL_BRANCH} https://gitlab.com/sfcgal/SFCGAL.git &&      cd SFCGAL &&      mkdir cmake-build &&      cd cmake-build &&      cmake -DCGAL_DIR=\"/src/CGAL\" -DCMAKE_PREFIX_PATH=/src/CGAL .. &&      make -j${BUILD_THREADS} &&      make install &&      cd /src && rm -rf SFCGAL" did not complete successfully: exit code: 1
------
 > [ 6/15] RUN git clone --depth 1 --branch master https://gitlab.com/sfcgal/SFCGAL.git &&      cd SFCGAL &&      mkdir cmake-build &&      cd cmake-build &&      cmake -DCGAL_DIR="/src/CGAL" -DCMAKE_PREFIX_PATH=/src/CGAL .. &&      make -j4 &&      make install &&      cd /src && rm -rf SFCGAL:
3.495     nlohmann_jsonConfig.cmake
3.495     nlohmann_json-config.cmake
3.495 
3.495   Add the installation prefix of "nlohmann_json" to CMAKE_PREFIX_PATH or set
3.495   "nlohmann_json_DIR" to a directory containing one of the above files.  If
3.495   "nlohmann_json" provides a separate development package or SDK, be sure it
3.495   has been installed.
3.495 
3.495 
3.496 -- Configuring incomplete, errors occurred!
------
Dockerfile:64
--------------------
  63 |     ARG SFCGAL_BRANCH=master
  64 | >>> RUN git clone --depth 1 --branch ${SFCGAL_BRANCH} https://gitlab.com/sfcgal/SFCGAL.git && \
  65 | >>>      cd SFCGAL && \
  66 | >>>      mkdir cmake-build && \
  67 | >>>      cd cmake-build && \
  68 | >>>      cmake -DCGAL_DIR="/src/CGAL" -DCMAKE_PREFIX_PATH=/src/CGAL .. && \
  69 | >>>      make -j${BUILD_THREADS} && \
  70 | >>>      make install && \
  71 | >>>      cd /src && rm -rf SFCGAL
  72 |     
--------------------
ERROR: failed to solve: process "/bin/sh -c git clone --depth 1 --branch ${SFCGAL_BRANCH} https://gitlab.com/sfcgal/SFCGAL.git &&      cd SFCGAL &&      mkdir cmake-build &&      cd cmake-build &&      cmake -DCGAL_DIR=\"/src/CGAL\" -DCMAKE_PREFIX_PATH=/src/CGAL .. &&      make -j${BUILD_THREADS} &&      make install &&      cd /src && rm -rf SFCGAL" did not complete successfully: exit code: 1
Env Batch selected: weekly [{'name': 'latest', 'PG': 'master', 'GEOS': 'main', 'GDAL': 'master', 'PROJ': 'master', 'PG_CC': 'gcc', 'SFCGAL': 'master'}, {'name': 'stable_pg18', 'PG': 'REL_18_STABLE', 'GEOS': 'main', 'GDAL': 'release/3.11', 'PROJ': '9.6', 'PG_CC': 'gcc', 'SFCGAL': 'v2.1.0'}, {'name': 'stable_pg17', 'PG': 'REL_17_STABLE', 'GEOS': '3.13', 'GDAL': 'release/3.9', 'PROJ': '9.4', 'PG_CC': 'gcc', 'SFCGAL': 'v2.1.0'}]
Traceback (most recent call last):
  File "/home/jenkins/workspace/PostGIS-build-env/label/docker/build.py", line 182, in <module>
    subprocess.check_call([
  File "/usr/lib/python3.9/subprocess.py", line 373, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['docker', 'build', '--pull', '--build-arg', 'BUILD_DATE=20251111', '--build-arg', 'POSTGRES_BRANCH=master', '--build-arg', 'GEOS_BRANCH=main', '--build-arg', 'GDAL_BRANCH=master', '--build-arg', 'PROJ_BRANCH=master', '--build-arg', 'PG_CC=gcc', '--build-arg', 'SFCGAL_BRANCH=master', '-t', 'postgis/postgis-build-env:latest', '.']' returned non-zero exit status 1.
Build step 'Execute shell' marked build as failure
[Gitea] do not publish assets due to build being non-Successfully
Finished: FAILURE
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -54,6 +54,8 @@ ARG BUILD_THREADS=4
 
 
 # nlohmann/json - header-only library for SFCGAL (with CMake support)
+# Build type defaults to Release; override with --build-arg if needed.
+ARG DOCKER_CMAKE_BUILD_TYPE=Release
 RUN set -ex \
     && mkdir -p /usr/src \
     && cd /usr/src \

